src/main/webapp/app/exercises/programming/manage/programming-exercise-detail.component.html

```diff
@@ -120,6 +120,10 @@ <h2><span jhiTranslate="artemisApp.programmingExercise.detail.title">Programming
                 <dd>
                     <span>{{ programmingExercise.allowOnlineEditor }}</span>
                 </dd>
+                <dt><span jhiTranslate="artemisApp.programmingExercise.noLocalSetup">No Local Setup</span></dt>
```

Comment:
This should only be shown in the DEV mode

Reply:
I deleted this part, because the value of the variable noLocalSetup isn't stored in the database and therefore will be never assigned